pilot/pkg/security/authn/utils/utils_test.go

```diff
@@ -232,6 +324,12 @@ func TestBuildInboundFilterChain(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			defaultCipherSuites := features.TLSInboundCipherSuites
```

Comment:
Tests look like they are validating the xDS output which is good. Should have some e2e coverage for this too.

Reply:
The e2e tests will be in separate PRs.